cypress/plugins/index.js

```diff
@@ -18,7 +18,7 @@ module.exports = (on, config) => {
   // `config` is the resolved Cypress config
   addMatchImageSnapshotPlugin(on, config);
   require('@cypress/code-coverage/task')(on, config);
-  on('file:preprocessor', require('@cypress/code-coverage/use-browserify-istanbul'));
```

Comment:
didn't realize we'd be able to just swap like this with no issue! I saw that browserify ceased shipping and gave up 😂